src/test/java/uk/ac/ebi/atlas/bioentity/properties/BioEntityCardModelFactoryIT.java

```diff
@@ -37,6 +38,7 @@ class BioEntityCardModelFactoryIT {
     @Inject
     private BioEntityCardModelFactory subject;
 
+    @Disabled
```

Comment:
I would use the `value`
```suggestion
    @Disabled(value="disabled by Upendra as part of Jenkins CI integration #268")
```